src/cli.ts

```
@@ -232,16 +232,20 @@ boards
-        console.log(JSON.stringify(board, null, 2));
+        console.log(JSON.stringify({ ...board, lists: Object.values(listsMap) }, null, 2));
       } else {
         console.log(`Board: ${board.name}`);
         console.log(`ID: ${board.id}`);
         console.log(`Created: ${new Date(board.createdAt).toLocaleString()}`);
-        if (board.lists?.length) {
+        const lists = Object.values(listsMap);
+        if (lists.length) {
           console.log('\nLists:');
-          board.lists.forEach(l => console.log(`  - ${l.name}`));
+          lists.forEach(l => console.log(`  - ${l.name}`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)